jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/MavenProjectPropertiesTest.java

```diff
@@ -352,207 +298,204 @@ public void testIsWarProject() {
-    Assert.assertEquals(0, MavenProjectProperties.getVersionFromString(""));
-    Assert.assertEquals(0, MavenProjectProperties.getVersionFromString("11abc"));
-    Assert.assertEquals(0, MavenProjectProperties.getVersionFromString("1.abc"));
+    assertThat(MavenProjectProperties.getVersionFromString("asdfasdf")).isEqualTo(0);
+    assertThat(MavenProjectProperties.getVersionFromString("")).isEqualTo(0);
+    assertThat(MavenProjectProperties.getVersionFromString("11abc")).isEqualTo(0);
+    assertThat(MavenProjectProperties.getVersionFromString("1.abc")).isEqualTo(0);
   }
 
   @Test
   public void testGetMajorJavaVersion_undefinedDefaultsTo6() {
-    Assert.assertEquals(6, mavenProjectProperties.getMajorJavaVersion());
+    assertThat(mavenProjectProperties.getMajorJavaVersion()).isEqualTo(6);
   }
 
   @Test
   public void testGetMajorJavaVersion_targetProperty() {
-    Mockito.when(mockMavenProperties.getProperty("maven.compiler.target")).thenReturn("1.8");
-    Assert.assertEquals(8, mavenProjectProperties.getMajorJavaVersion());
+    when(mockMavenProperties.getProperty("maven.compiler.target")).thenReturn("1.8");
+    assertThat(mavenProjectProperties.getMajorJavaVersion()).isEqualTo(8);
 
-    Mockito.when(mockMavenProperties.getProperty("maven.compiler.target")).thenReturn("1.7");
-    Assert.assertEquals(7, mavenProjectProperties.getMajorJavaVersion());
+    when(mockMavenProperties.getProperty("maven.compiler.target")).thenReturn("1.7");
+    assertThat(mavenProjectProperties.getMajorJavaVersion()).isEqualTo(7);
 
-    Mockito.when(mockMavenProperties.getProperty("maven.compiler.target")).thenReturn("11");
-    Assert.assertEquals(11, mavenProjectProperties.getMajorJavaVersion());
+    when(mockMavenProperties.getProperty("maven.compiler.target")).thenReturn("11");
+    assertThat(mavenProjectProperties.getMajorJavaVersion()).isEqualTo(11);
   }
 
   @Test
   public void testValidateBaseImageVersion_releaseProperty() {
-    Mockito.when(mockMavenProperties.getProperty("maven.compiler.release")).thenReturn("1.8");
-    Assert.assertEquals(8, mavenProjectProperties.getMajorJavaVersion());
+    when(mockMavenProperties.getProperty("maven.compiler.release")).thenReturn("1.8");
+    assertThat(mavenProjectProperties.getMajorJavaVersion()).isEqualTo(8);
 
-    Mockito.when(mockMavenProperties.getProperty("maven.compiler.release")).thenReturn("1.7");
-    Assert.assertEquals(7, mavenProjectProperties.getMajorJavaVersion());
+    when(mockMavenProperties.getProperty("maven.compiler.release")).thenReturn("1.7");
+    assertThat(mavenProjectProperties.getMajorJavaVersion()).isEqualTo(7);
 
-    Mockito.when(mockMavenProperties.getProperty("maven.compiler.release")).thenReturn("9");
-    Assert.assertEquals(9, mavenProjectProperties.getMajorJavaVersion());
+    when(mockMavenProperties.getProperty("maven.compiler.release")).thenReturn("9");
+    assertThat(mavenProjectProperties.getMajorJavaVersion()).isEqualTo(9);
   }
```

Comment:
Um... I think this is reasonable. This file has 54 tests and >1000 lines of code, if that matters :sweat:

Reply:
Ah that's fair. Hm it looks like this test file is getting pretty long which might get harder to maintain as we keep adding stuff. It might be worth splitting this test file into smaller files (have one for tests for jars and another for wars, for example). Not in this PR but something to consider in the future.